lib/mods/replace-get-set.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const nodeTypes = require('ts-morph').TypeGuards;
+const {updateImports} = require('../imports.js');
+const {
+  isGlobal,
+  getOriginal,
+} = require('../originals-helper.js');
+const {formatCall} = require('./helpers.js');
+
+/**
+ * Replaces accessor calls in the current node with the Get-Originals version
+ *
+ * @param {Node} node - a ts-morph AST Node
+ * @param {Object} imports - the current list of imports
+ */
+function replaceGlobalGetSetCalls(node, imports) {
```

Comment:
Nit: naming is all over the place. In the comments it's accessor calls. In the function name it's global get set calls. In the filename and the export of index.js it's get set. Pick one :). I like get set or maybe "gets and sets".

Reply:
I'm just gonna go ahead and use `getset`, I stumbled on the name accessor calls late in the game so I added a few of those, but it's somewhat less explicit